app/views/layouts/landing_page_layout.html.erb

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>
-    Help for early years providers
+    Help for early years providers - UK.GOV
```

Comment:
1. Technically to match the service manual standard this should be: Help for early years providers - Help for early years providers - GOV.UK
2. Note GOV.UK and not typo of UK.GOV

Reply:
I'll change GOV.UK

But repeating Help for early years providers - Help for early years providers 

is not a good idea.  Its for Google Analytics and saying the same thing twice is not helpful

Reply:
Within jira story the attachment shows 'Department for Education' after 'Help for early years providers' within landing_page_layout.html

Reply:
Text in page titles should not be repeated

It does not make sense to Google Analytics

Reply:
The Jira story is wrong